src/SharedAssemblyInfo.cs

```diff
@@ -1,8 +1,16 @@
-﻿// <auto-generated/>
+﻿//-----------------------------------------------------------------------
+// <copyright file="SharedAssemblyInfo.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
```

Comment:
This looks like one of the old headers, everywhere else we already set 2016 as year and Lightbend as company name